lib/prh-service.yml

```
@@ -154,3 +154,49 @@ rules:
     specs:
       - from: つくる、つむぐ、つづく
         to: つくる、つむぐ、つづく、
+        
+  - expected: 次のページ
+    patterns:
```

```suggestion
    patterns: /次(の)?ページ(へ)?/
```

にして

```
      - 次ページ
      - 次ページへ
      - 次のページへ
```

は削除でよさそうな気がします。